packages/playwright/src/EmbeddedResourceProvider.cs

```diff
@@ -0,0 +1,19 @@
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deque.AxeCore.Playwright
+{
+    internal static class EmbeddedResourceProvider
+    {
+        public static Task<string> ReadEmbeddedFileAsync(string fileName)
+        {
+            var assembly = typeof(EmbeddedResourceProvider).Assembly;
+            var resourceStream = assembly.GetManifestResourceStream($"Deque.AxeCore.Playwright.Resources.{fileName}");
+            using (var reader = new StreamReader(resourceStream, Encoding.UTF8))
+            {
+                return reader.ReadToEndAsync();
```

Comment:
This needs to be an `async` method that uses `return await reader.ReadToEndAsync().ConfigureAwait(false);` here, rather than propagating the `Task` returned by `ReadToEndAsync` as-is.

Normally, prefering `await` to returning sub-tasks directly would just be a nice-to-have for the purposes of improving call stacks for debugging purposes. [This blog post explains the difference in exception stack trace output](https://vkontech.com/exploring-the-async-await-state-machine-stack-traces-and-refactoring-pitfalls/). (This difference also happens in TypeScript/JavaScript, for essentially the same reasons).

However, in this case, it's not just about stack trace usability, it's also required for correctness - as-written, this code has a use-after-dispose bug because the `using` statement responsible for the `StreamReader`'s lifetime is closed before the `ReadToEndAsync()` task is necessarily done with its work.